pwa-devdocs/src/pwa-buildpack/reference/targetables/TargetableESModule/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: TargetableESModule
+---
+
+<!--
+The reference doc content is generated automatically from the source code.
+To update this section, update the doc blocks in the source code
+-->
+
+{% include auto-generated/pwa-buildpack/lib/WebpackTools/targetables/TargetableESModule.md %}
+
+## Examples
+
+Code examples for the `TargetableESModule` class.
+
+### Add and use import statements
+
+Calling the `addImport()` function returns a [`SingleImportStatement`][] object.
+Use this object to refer to the component in your code.
+
+```jsx
+const logger = esModule.addImport('import logger from "./logger"');
+esModule.insertAfterSource("./logger';\n", `${logger.binding}('startup')`)
+```
+
+The `SingleImportStatement` class overrides its `toString()` method to return the value of its `.binding` property,
+so you can use the object itself in your templates
+
+```jsx
+const logger = esModule.addImport('import logger from "./logger"');
+esModule.insertAfterSource("./logger';\n", `${logger}('startup')`)
+```
+
+#### Import statement limits
+
+The `addImport()` function can only handle import statements with a single binding.
```

Comment:
It's nice that you have mentioned this. Do we have a plan to handle/support multiple imports? If so, I guess it makes sense to mention that here for curious devs.

Reply:
I'm not aware of any plans